### DISCUSSION TITLE
Step2   blank tft for Lovelace

### DISCUSSION THREAD

**Discussion by MarkuZP** [Blackymas/NSPanel_HA_Blueprint#142] [Q&A]
Super Projekt!
 Ich habe eine Frage. Unter Punkt 3, wo beschrieben wird wie man die leere TFT Datei hochlädt wegen Lovelace heisst es:
"..gehe jetzt zu dem Punkt "Entwicklerwerzeuge --> Dienste" und suche dort nach dem Dienst  
"ESPHome: DEVICE_NAME_upload_tft" --> (esphome.device_name_upload_tft)"  Diese Dienst finde ich noch gar nicht. Aber wenn ich ERST ESP Home auf das NSPanel flashe, dann ist ja kein Lovelace mehr drauf. Das kapiere ich nicht....

**Comment by Blackymas** [comment]
Der Kollege hat es damals genau so gemacht. Er hat nur die ESPHome Version von Lovelace installiert danach kannst du das easy machen.

Ich schau mal das ich Ihn nochmal frage.

Soweit ich weiß ist aber bei Lovelace auch dokumentiert wie du zb die beta tft hochladen kannst. Theoretisch musst du nur das mit meiner Blank TFT machen

**Reply by MarkuZP** [reply]
Hi Blacky! Zunächst möchte Ich mich bei Dir bedanken für die grossartige Arbeit, die Du da abgeliefert hast! Das ist genau die GUI, die ich mir gewünscht hatte für das NS Panel. Lovelace ist auch nicht schlecht aber etwas weniger ergonomisch und stylish als Deine Lösung. Mega vielen Dank auch für den Support!!! 
Ich bin noch eher asl Noob hier unterwegs deshalb hatte ich es vielleicht nicht ganz verstanden weil wenn man die Steps der Anleitung genau befolgt aber z.B. Tasmota auf dem Panel läuft, dann geht das halt nicht. Das hab ich jetzt verstanden und mit Tasmota das blank tft geflasht. Das Bild ist weiß! 
Nach dem Flashen mit dem ESP Home Projekt wie in der Anleitung beschrieben hab ich jetzt aber das Problem, dass dieses neu geflashte ESPHome-device nicht gefunden wird in HA unter Integrationen. Ich denke die DNS/ IP Auflösung geht irgendwie nicht.
Hast Du ne Idee?

<img width="1072" alt="Bildschirm­foto 2023-01-02 um 11 37 13" src="https://user-images.githubusercontent.com/109298227/210220470-255bd817-e41a-4719-a57a-e344c4a509f2.png">

**Reply by MarkuZP** [reply]
Also das ESP Home device ist immer noch offline. Hab schon  mehrfach neu geflasht ohne Erfolg.
<img width="666" alt="Bildschirm­foto 2023-01-02 um 11 41 42" src="https://user-images.githubusercontent.com/109298227/210220947-842017a7-6473-4ed8-9b43-46a34fa09737.png">

**Comment by MarkuZP** [comment]
Habs jetzt noch hinbekommen! Danke!

**Reply by Blackymas** [reply]
wäre bombe von dir wenn du deine Lösung noch hier schreiben könntest was es war - damit andere daraus ihre Fehler beheben könnten ;)

und danke fürs Feedback

**Comment by DerBo19** [comment]
Hallo Blackymas,

zuerst´: VIELEN HERZLICHEN DANK FÜR DIE UNENDLICHE MÜHE, welche du dir bei der Umsetzung und Betreuung gibst. Es kann nicht überschätzt werden, wie viel solche Projekte uns allen bringen. Nicht zuletzt befreit es uns aus den Fängen der All-In-One-Mafia und ihren aufgezwungenen Hubs.

Jetzt aber zum Grund meiner Nachricht: Ich hatte mit demselben Problem hier zu kämpfen wie der TE. Ich bin eigentlich ein ziemlicher Anfänger ohne Programmierungskenntnisse, konnte aber das Problem lösen:

Wer vorher Lovelance UI mit Tasmota drauf gespielt hatte und diesen Dienst bei den Entwicklungstools nicht finden kann, kann einfach mit dem Tasmota Web Installer nochmals die Tasmota32-ns-panel-Firmware draufgespielt. Danach bin ich verfahren wie in der Kurzanleitung. Das hat geklappt. Ich konnte in ESP-Home das Device auch installieren. Allerdings hänge ich jetzt an der TFT-Datei. Es gibt nicht, wie in den einschlägigen Videos, einen Schalter zum switchen sondern es steht nur "DRÜCKEN" hinter der Option Update TFT-display. Ich erkenne also nicht, wann und ob das abgeschlossen ist. Das Netzwerk ist in der factory.bin eingerichtet und auch die lokale Adresse mit "[MEINE-IP]:8123/local/nspanel_eu.tft" angegeben. Über den Ladebildschirm im Device komme ich aber nicht hinaus... Allerdings macht der Bildschirm mich stutzig. Der sieht nämlich genauso aus wie bei meiner Lovelance UI (runder blauer Kreis mit "Waiting for Content".

Würde mich trotzdem interessieren, wo und wie ich dieses Entwicklungstool aktivieren kann.
